ucm2/rockchip_es8316/HiFi.conf

```diff
@@ -0,0 +1,102 @@
+# Reference:
+# https://gitlab.manjaro.org/manjaro-arm/packages/community/pinebookpro-post-install/-/blob/4b4206c57b21044bf9c00848bd1f8b6f710d19eb/asound.state
+# https://gitlab.manjaro.org/manjaro-arm/packages/community/pinebookpro-audio/-/blob/1f567db09495acc3beb5405c9ef85c3c07b86785/audio_jack_plugged_in.sh
+
+SectionVerb {
+	Value {
+		MinBufferLevel "512"
+	}
+
+	EnableSequence [
+		cset "name='Speaker Switch' off"
+		cset "name='Headphone Playback Volume' 3"
+		cset "name='Headphone Mixer Volume' 11"
+		cset "name='Playback Polarity' 'R Invert'"
+		cset "name='DAC Playback Volume' 192"
+		cset "name='DAC Soft Ramp Switch' off"
+		cset "name='DAC Soft Ramp Rate' 4"
+		cset "name='DAC Notch Filter Switch' off"
+		cset "name='DAC Double Fs Switch' off"
+		cset "name='DAC Stereo Enhancement' 5"
+		cset "name='DAC Mono Mix Switch' off"
+		cset "name='Capture Polarity' Normal"
+		cset "name='Mic Boost Switch' on"
+		cset "name='ADC Capture Volume' 192"
+		cset "name='ADC PGA Gain Volume' 7"
+		cset "name='ADC Soft Ramp Switch' off"
+		cset "name='ADC Double Fs Switch' off"
+		cset "name='ALC Capture Switch' off"
+		cset "name='ALC Capture Max Volume' 28"
+		cset "name='ALC Capture Min Volume' 0"
+		cset "name='ALC Capture Target Volume' 10"
+		cset "name='ALC Capture Hold Time' 0"
+		cset "name='ALC Capture Decay Time' 3"
+		cset "name='ALC Capture Attack Time' 2"
+		cset "name='ALC Capture Noise Gate Switch' off"
+		cset "name='ALC Capture Noise Gate Threshold' 0"
+		cset "name='ALC Capture Noise Gate Type' 'Constant PGA Gain'"
+		cset "name='Differential Mux' lin1-rin1"
+		cset "name='Digital Mic Mux' 'dmic disable'"
+		cset "name='DAC Source Mux' 'LDATA TO LDAC, RDATA TO RDAC'"
+		cset "name='Left Headphone Mux' lin1-rin1"
+		cset "name='Right Headphone Mux' lin1-rin1"
+		cset "name='Left Headphone Mixer LLIN Switch' off"
+		cset "name='Left Headphone Mixer Left DAC Switch' on"
+		cset "name='Right Headphone Mixer RLIN Switch' off"
+		cset "name='Right Headphone Mixer Right DAC Switch' on"
+	]
+}
```

Comment:
The default card settings should go to the BootSequence or FixedBootSequence. The verb sequence should set only path / controls used in the device settings.

Reply:
Sure!  Move it to the BootSequence.